microprofile-opentracing/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.jboss.eap.qe</groupId>
+        <artifactId>microprofile-test-suite</artifactId>
+        <version>1.0.0.Final-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>microprofile-opentracing</artifactId>
+
+    <!--Port of:
+        https://github.com/rhoar-qe/thorntail-test-suite/tree/master/microprofile/microprofile-opentracing-1.0
+        https://github.com/rhoar-qe/thorntail-test-suite/tree/master/microprofile/microprofile-opentracing-1.3
+    -->
+
+    <dependencies>
+        <dependency>
+            <groupId>org.eclipse.microprofile</groupId>
+            <artifactId>microprofile</artifactId>
+            <type>pom</type>
+        </dependency>
+        <dependency>
+            <groupId>io.jaegertracing</groupId>
+            <artifactId>jaeger-client</artifactId>
+            <version>${version.io.jaegertracing.jaeger-client}</version>
```

Comment:
Could you add all dependencies to parent pom to dependency management (if not already there), please?

Reply:
Fixed, thanks.